frontend/src/pages/Home.jsx

```diff
@@ -1,14 +1,19 @@
 import React from "react";
 import Navbar from "../components/Header/Navbar";
 import HeroSection from "../components/Home/HeroSection";
+import Div1 from "../components/AboutUs/Div1";
+import Div2 from "../components/AboutUs/Div2";
+import Footer from "../components/Footer/Footer";
 
 const Home = () => {
   return (
     <div>
-      <Navbar/>
-        <HeroSection>HeroSection</HeroSection>
+      <Navbar />
+      <HeroSection>HeroSection</HeroSection>
+      <Div2/>
+      <Div1/>
```

Comment:
I don't like these namings. Components should have description names em. If I look at this code, I have no idea what those divs contain.

Reply:
Thank you for your feedback!
I'm gonna edit it right now